config/features.yml

```diff
@@ -185,7 +185,11 @@ features:
     description: Send direct deposit emails using VaNotify service
   disability_hypertension_compensation_fast_track:
     actor_type: user
-    description: Fast tracks 526 disability compensation claims by submitting additional health data
+    description: Fast tracks 526 disability compensation hypertension claims by submitting additional health data
+    enable_in_development: true
+  disability_asthma_compensation_fast_track:
```

Comment:
I'd like to start renaming these Flipper keys to `rrd_#{disability}_compensation`
(as noted [here](https://github.com/department-of-veterans-affairs/vets-api/blob/d9e3a05a2e03a5089b18aba1bf9119f4d1466397/app/services/rapid_ready_for_decision/processor_selector.rb#L41)), so we can then add other RRD-specific Flipper keys in one contiguous section, where all entries begin with prefix `rrd_`.
Should we start doing it in this PR? 
I can add `rrd_hypertension_compensation` in a separate PR and enable it in prod (along with code changes to look for both Flipper flags), and then follow-up with another PR to remove `disability_hypertension_compensation_fast_track`.

Reply:
Sure thing. Since it's already programmatically constructed in ProcessorSelector, I'll just change the asthma-related instances where it's "hard-coded" and let #9360 handle the rest.

(Unless #9360 merges first, in which case I'll rebase and update everything here to the New Way)